tests/data/taca_test_cfg.yaml

```diff
@@ -0,0 +1,89 @@
+log:
+    log_file: data/taca.log
+
+storage:
+    data_dirs:
+        - /srv/illumina/hiseq_data
+        - /srv/illumina/miseq_data
+    archive_dirs:
+        - /srv/illumina/hiseq_data/nosync
+        - /srv/illumina/miseq_data/nosync
+    irods:
+        irodsHome: /ssUppnexZone/proj/a2010002
+
+cleanup:
```

Comment:
@guillermo-carrasco : hej, I don't think these will be useful for tests. You can remove them, I will update with proper test paths when I write tests :)